<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiTensorFieldExplorerClass</class>
 <widget class="QMainWindow" name="MultiTensorFieldExplorerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiTensorFieldExplorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLViewer" name="glViewer" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="showSlicesRecovered_edit">
    <property name="geometry">
     <rect>
      <x>681</x>
      <y>537</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showPddsRecovered_edit">
    <property name="geometry">
     <rect>
      <x>681</x>
      <y>557</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0 1 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="showSlices_label">
    <property name="geometry">
     <rect>
      <x>613</x>
      <y>537</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show slices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="showPdds_label">
    <property name="geometry">
     <rect>
      <x>613</x>
      <y>560</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show pdds:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateRecovered_button">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>514</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runDBF">
    <property name="geometry">
     <rect>
      <x>1036</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run DBF</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="visualizationType_radiogroup">
    <property name="geometry">
     <rect>
      <x>1035</x>
      <y>80</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Visualization type</string>
    </property>
    <widget class="QRadioButton" name="orientation_radio">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>20</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Orientation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="sizeCompartment_radio">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>40</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Size compartment</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="error_radio">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>60</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="sparcity_radio">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>80</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Sparcity</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="arrows_radio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Arrows</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="apparentDiffusion_radio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Apparent Diffusion</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="orientationDiffusion_radio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Orientation Diffusion</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="clusterColors_radio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Cluster colors</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loadOnlyGT_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>514</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load only</string>
    </property>
   </widget>
   <widget class="GLViewer" name="glViewerGT" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateGT_button">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>514</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show slices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Show pdds:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showSlicesGT_edit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showPddsGT_edit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0 1 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sliceGT_edit">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>540</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rowGT_edit">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>560</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>21</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="columnGT_edit">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>580</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Row:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>580</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Column:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectGT_button">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>514</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadOnlyRecovered_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>514</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load only</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectRecovered_button">
    <property name="geometry">
     <rect>
      <x>792</x>
      <y>514</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Slice:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>560</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Row:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>580</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Column:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sliceRecovered_edit">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>540</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rowRecovered_edit">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>560</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>21</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="columnRecovered_edit">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>580</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1044</x>
      <y>297</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Big. P. Thr:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1046</x>
      <y>319</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Neigh. S:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fitWithNeighbors_check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>378</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit with neighbors</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="addGDTIPdd_check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>398</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add HARDI PDD</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="regularizeIterative_check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>418</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Regularize (Iterative)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="useFAIndicator_check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>438</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use FA indicator</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="useGDTIPostProc_check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>458</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use HARDI post-proc</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1046</x>
      <y>277</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>SNR:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bigPeaksThr_edit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>295</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="neighSize_edit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>315</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SNR_edit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>275</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>339</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sampling:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sampling_edit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>335</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="applyTensorSplitting_check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>478</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply tensor splitting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="random_button">
    <property name="geometry">
     <rect>
      <x>1115</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1115</x>
      <y>563</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Target:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="target_edit">
    <property name="geometry">
     <rect>
      <x>1155</x>
      <y>560</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="evaluate_button">
    <property name="geometry">
     <rect>
      <x>1036</x>
      <y>26</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compare_button">
    <property name="geometry">
     <rect>
      <x>1036</x>
      <y>2</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buildTrainingSet_button">
    <property name="geometry">
     <rect>
      <x>1155</x>
      <y>583</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BuildTrainingSet</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="iterateDiffProf_check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>498</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterate Diff. Prof.</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="testDenoising_button">
    <property name="geometry">
     <rect>
      <x>1105</x>
      <y>613</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test denoising</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="spatialSmoothing_check">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>241</y>
      <width>111</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Spatial smoothing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1046</x>
      <y>261</y>
      <width>46</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Lambda:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lambda_edit">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>258</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0.9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showGroupColors_check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>360</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Show group colors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alphaBehaviorStudy_button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha-study</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>264</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>284</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="alpha0_edit">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>260</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="alpha1_edit">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>280</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.15</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spectralClustering_button">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>474</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Spectral Cl.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="d0_edit">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>350</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="theta0_edit">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>370</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deltaTheta0_edit">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>390</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>350</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>d0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>1174</x>
      <y>370</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Theta0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>390</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>dTheta0:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveEmbedded_check">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>434</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Save embedded</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="loadClustering_check">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>450</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Load clustering</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="loadEmbedded_check">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>420</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Load embedded</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>510</y>
      <width>120</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Save</string>
    </property>
    <widget class="QPushButton" name="save_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="slice_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Slice</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="volume_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>82</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="mds_button">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>500</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>MDS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testMask_button">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test mask</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gtFromPaths_button">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GT from Paths</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mtfToLCI">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>MTF to LCI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LCIToMask">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>24</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LCI to Mask</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLViewer</class>
   <extends>QWidget</extends>
   <header>glviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MultiTensorFieldExplorer.qrc"/>
 </resources>
 <connections/>
</ui>
